cmd/brew-test-bot.rb

```diff
@@ -1561,6 +1562,8 @@ def test_ci_upload(tap)
           unless package_exists
             package_blob = <<~EOS
               {"name": "#{bintray_package}",
+               "vcs_url": "#{bintray_bottle_url}",
+               "licenses": ["MIT", "GPL-2.0", "GPL-3.0", "BSD"],
```

Comment:
These two lines aren't correct so I'm a bit dubious about having them hardcoded.

Reply:
Yea, I know, but these fields are required to create a package on bintray as normal user, so what can we fill them with?

Reply:
I'm not sure but if they are required fields I don't think Bintray will look kindly at us setting them by default to incorrect values.

Reply:
I suggest…
```
               "vcs_url": "https://github.com/Homebrew/brew",
               "licenses": ["BSD-2"],
```